tutorials/build_system/deployment.md

```diff
@@ -49,7 +62,9 @@ The gradle-intellij-plugin provides a number of [configuration options](https://
  The Gradle plugin will also update the `<idea-version since-build=.../>` values within the `plugin.xml` file to match the `intellij.version`, valid until the last release in the current major version, however you can disable this feature by setting the `intellij.updateSinceUntilBuild` option to `false`.
 
 ```groovy
-apply plugin: 'org.jetbrains.intellij'
+plugins {
+    id 'org.jetbrains.intellij' version '0.4.2'
```

Comment:
there is 0.4.5 now that is able to work with java 11

Reply:
Good catch, of course no reason to put an old version there, although I assume it is not intended to update this on every new release. Will change it.

Reply:
yes, I think no need to update it on every release. Maybe add a note that it makes sense to check whether the newer version is available at https://plugins.gradle.org/plugin/org.jetbrains.intellij or subscribe to releases at https://github.com/JetBrains/gradle-intellij-plugin/releases?

Reply:
Yeah I was thinking about that too, I'll do that.